components/matchBox/TeamNames.tsx

```diff
@@ -3,12 +3,20 @@ import styles from "./MatchBox.module.css";
 export type Teams = {
   home: string;
   away: string;
+  handlePick: any;
```

Comment:
I am grateful for tips on which type this is. Since I have not been able to find the right one so far, I have temporarily entered "any" 🤹

Reply:
not sure but maybe a `void` type?